Small additions/edits to ruler and alertmanager docs (#2983)
grafana/mimir: docs/sources/operators-guide/tools/mimirtool.md

The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.

> > **Note:** The Mimir Alertmanager exposes the configuration API according to the path set by the `-server.path-prefix` flag. It does not use the path set by the `-http.alertmanager-http-prefix` flag.

Reviewer: ```suggestion
> **Note:** The Mimir Alertmanager exposes the configuration API based on the path that you set via the `-server.path-prefix` flag; it does not use the path of the `-http.alertmanager-http-prefix` flag.
```
I am not sure if the user has set both of these paths or one of them. Therefore, I changed this to possession (the path of the ...).

> > With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion.

Reviewer 2: I would try to clarify it. I don't know if my edit is better, but I would try to make it easier to understand.

```suggestion
> With the default configuration of `-server.path-prefix`, `mimirtool` `--address` flag should be set to an URL with the `/` path.
```

Author: I tried thinking of something more clear, but I couldn't do much better.

what do you think about just giving an example?
```suggestion
> With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion: for example `http://alertmanager.mimir.example.com/`.
```

Reviewer 2: The example could help. @osg-grafana could help further :)

> > With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion.

Reviewer: ```suggestion
> If you run Mimir with the default configuration (`-server.path-prefix`, where the default value is `/`), then only set the hostname for the `mimirtool --address` flag; do not set a path-specific address. For example, `/` is correct, and `/alertmanager` is incorrect.
```

Author: we've used "`mimirtool` `--address` flag" in another article (alertmanager.md), should I update it there too?

- Interact with individual rule groups in the Mimir ruler
- Manipulate local rule files

Some of the functionality that `mimirtool rules` offers is also available as a GitHub Action.

> For more information, refer to the [documentation of the action](https://github.com/grafana/mimir/blob/main/operations/mimir-rules-action/README.md).

Reviewer 2: ```suggestion
For more information, refer to the [documentation of Mimirtool Github Action](https://github.com/grafana/mimir/blob/main/operations/mimir-rules-action/README.md).
```

> For more information, refer to the [documentation of Mimirtool Github Action](https://github.com/grafana/mimir/blob/main/operations/mimir-rules-action/README.md).

Author: it cant be relref since we don't publish that readme anywhere
